filebeat/tests/system/test_tcp.py

```diff
@@ -0,0 +1,68 @@
+from filebeat import BaseTest
+import socket
+
+
+class Test(BaseTest):
```

Comment:
Nice tests. I already head in mind to ask for this, especially seeing if the shut down works 👍